docs/docs/user_guide/account_structure.md

```diff
@@ -6,19 +6,17 @@ sidebar_label: Account Structure
 
 ## Account Structure
 
-Service Catalog uses three kinds of accounts, whose names are used in this guide.  *Master* and *Member* accounts are terms referring to [AWS Organizations](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_introduction.html), while *Main* account is a Service Catalog term.
+Service Catalog uses three kinds of accounts, whose names are used in this guide. _Master_ and _Member_ accounts are terms referring to [AWS Organizations](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_introduction.html), while _Main_ account is a Service Catalog term.
 
-* **Main**: The account within which Service Catalog is deployed.  The main account will be billed for all AWS usage charges for the Service Catalog deployment itself: the S3 bucket holding the website, and other resources not created by a user.  Service Catalog may be deployed in a master account (the account holding the Organization), or a member account (within the Organization).  In either case, this account is called the Service Catalog *Main* account.
-* **Master**: The account hosting the AWS Organization. The master account is responsible for the billing of the member accounts within the Organization.
-* **Member**: An account within an AWS Organization.  When you create an account using [Create AWS Account](/deployment/post_deployment/aws_accounts#Create_AWS_Account), that account is created as a member of the Organization.
+- **Main**: The account within which Service Catalog is deployed. The main account will be billed for all AWS usage charges for the Service Catalog deployment itself: the S3 bucket holding the website, and other resources not created by a user. Service Catalog may be deployed in a master account (the account holding the Organization), or a member account (within the Organization). In either case, this account is called the Service Catalog _Main_ account.
+- **Master**: The account hosting the AWS Organization. The master account is responsible for the billing of the member accounts within the Organization.
+- **Member**: An account within an AWS Organization. When you create an account using [Create AWS Account](/deployment/post_deployment/aws_accounts#Create_AWS_Account), that account is created as a member of the Organization.
```

Comment:
When I click 'Create AWS Account' after viewing the file, it takes me here[1] to some page that doesn't exist. Is that expected?

[1] https://github.com/awslabs/service-workbench-on-aws/blob/37ee47cfdef4b662b7f2b929e7dfe99e373fa230/docs/docs/user_guide/account_structure.md

Reply:
Nope looks like another dead link :(

Reply:
Comment can be resolved since it would be fixed in this PR: https://github.com/awslabs/service-workbench-on-aws/pull/219